### ISSUE TITLE
Hook up the default command for driving.

### ISSUE THREAD

**Issue by drewwhis** [frc538/2020-infinite-recharge#6]

**Comment by drewwhis** [comment]
This depends on #5

**Comment by drewwhis** [comment]
When we create commands, there are a few steps we want to follow. These are slightly different for default commands, so we'll look at regular commands on a separate task (#14)

**Comment by drewwhis** [comment]
To set up a default command, we want to do that in the constructor of the RobotContainer. For this particular command, we want to make sure the driver control has been created, first.

0. Create the driver controller. For example, `private final Joystick mDriverOneController = new Joystick(0);`.
1. Reference the subsystem to set its default command. In this case, `mDriveSubsystem.setDefaultCommand();`.
2. Inside the parentheses, we need to create the command. In this case, `mDriveSubsystem.setDefaultCommand(new DriveCommand());`.
3. For the drive command to have access to the subsystem and the joystick, we need to pass those values in. In this case, `mDriveSubsystem.setDefaultCommand(new DriveCommand(drive, joystick));`.

A similar process would be followed for any other subsystem that needs a default command. The default command runs when **no other command** is using the subsystem.

One key for a default command vs. other commands is that the default command's `isFinished` should always return false.

**Comment by drewwhis** [comment]
Breanna and Maison knocked this out on Saturday.
